### PR TITLE
Created a new reactor type MBSampledReactor for modeling laser lab Mass-Spec experiments

### PR DESCRIPTION
**Pull request by jimchu10**
### Motivation or Problem
To model the sampling delay in mass-spec of Green group laser apparatus, it is important to generate a new reactor type.

### Description of Changes
New reactor type added, and can be used in simulate.py

### Testing
The MBSampledReactor added works same as what it was before rebase on master. Same time-dependent concentration for species by giving same input files

### CODE REVIEW

**Comment by mliu49** [comment]
I think it would make sense for MBSampledReactor to be a subclass of SimpleReactor. I compared the two, and it seemed like only the `residual` method was changed. Doing it this way would make it much easier to maintain.

I added a commit making this change, but have not tested it yet beyond checking that it compiled corrected. Could you run a simulation to see if this works and gives the same result as before?

**Comment by mliu49** [comment]
Update regarding my previous commit. Turns out that it won't work because SimpleReactor defines a `jacobian` method which necessarily cannot be included for MBSampledReactor.

File: rmgpy/tools/simulateTest.py [hunk 1]
```diff
@@ -74,6 +76,23 @@ def test_liquid(self):
         shutil.rmtree(os.path.join(folder, 'solver'))
         os.remove(os.path.join(folder, 'simulate.log'))
 
+    def test_mbSampled(self):
+        """Test that we can simulate an MBSampledReactor"""
+        folder = os.path.join(os.path.dirname(rmgpy.__file__), 'tools/data/sim/mbSampled')
```
Comment (amarkpayne) [inline rmgpy/tools/simulateTest.py]: Need to join tools, data, sim, and mbSampled. Can't use `/`

**Comment by amarkpayne** [comment]
When I run the test locally, this is the plot that I get. Is this what the simulation is supposed to do?

simulation_1_30

File: rmgpy/tools/simulateTest.py (hunk 1 above)
Comment (mliu49) [reply rmgpy/tools/simulateTest.py]: Thanks for keeping up with Windows support 😉

**Comment by mliu49** [comment]
Yeah, the concentrations are all extremely low, so they don't show up on the plot.

File: rmgpy/tools/simulateTest.py (hunk 1 above)
Comment (amarkpayne) [reply rmgpy/tools/simulateTest.py]: RMG-Py: potentially Windows compatible since 2019 :)

**Pull request merged**